qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1094,6 +1095,79 @@ def categories(self):
 
         return cols
 
+    def update(self, md_template):
+        r"""Update values in the sample template
+
+        Parameters
+        ----------
+        md_template : DataFrame
+            The metadata template file contents indexed by samples Ids
+
+        Raises
+        ------
+        QiitaDBError
+            If md_template and db do not have the same sample ids
+            If md_template and db do not have the same column headers
+            If self.can_be_updated is not True
+        """
+        self._check_subclass()
```

Comment:
Agree

Doesn't that imply the classmethods shouldn't actually be classmethods as they depend on instantiated state?